.github/workflows/update-docker-requirements.yml

```diff
@@ -56,10 +69,8 @@ jobs:
           # prettier-ignore
           body:
             https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}
-          reviewers: ${{ github.actor }}
+          reviewers: ${{ env.pr_author }},${{ env.pr_mergedBy }}
 
       - name: Show diff
-        # prettier-ignore
-        if:
-          ${{ steps.update.outputs.needs-update && github.event_name != 'workflow_dispatch' }}
+        if: ${{ steps.update.outputs.needs-update }}
```

Comment:
This whole step can be removed as we either get PR or we don't. No one will look at the action run anyway.